factory.go

```diff
@@ -134,6 +145,10 @@ func (f *LoggerFactory) setDefaultFormat() error {
 	)
 }
 
+func (f *LoggerFactory) setHook(l *logrus.Logger) {
+	l.AddHook(filename.NewHook(logrus.DebugLevel))
```

Comment:
If I am not wrong, this will be triggered by DEBUG messages. What I requesteds is include the filename to all kind of messages when the level is set to debug.